### PR TITLE
Correct the example for sub - initialEvent and error MULTIEVENT_SUBSCRIPTION_NOT_SUPPORTED and some typos.

### LINKED ISSUE #266: [geofencing-subscriptions]: MultieventSubscriptionNotSupported instead of PermissionDenied.
**Problem description**
The example for CreateSubscriptionUnprocessableEntity422 - MultieventSubscriptionNotSupported instead of PermissionDenied.

```
MultieventSubscriptionNotSupported:
  value:
    status: 422
      code: MULTIEVENT_SUBSCRIPTION_NOT_SUPPORTED
      message: Multi event types subscription not managed
```

### PR DESCRIPTION
**Pull request by dfischer-tech**
#### What type of PR is this?

Add one of the following kinds:
* correction

#### What this PR does / why we need it:
Correct the example and some typos.

#### Which issue(s) this PR fixes:

Fixes #266

### CODE REVIEW

**Review by maxl2287 — DISMISSED** [review]
LGTM! 
Thanks @dfischer-tech! 🚀

File: code/API_definitions/geofencing-subscriptions.yaml [hunk 1]
```diff
@@ -372,7 +372,7 @@ components:
           $ref: "#/components/schemas/SinkCredential"
         types:
           description: |
-            Camara Event types eligible to be delivered by this subscription.
+            Camara Event types are eligible to be delivered by this subscription.
```
Comment (jlurien) [inline code/API_definitions/geofencing-subscriptions.yaml:375]:
```suggestion
            Camara Event types which are eligible to be delivered by this subscription.
```

**Review by jlurien — DISMISSED** [review]

**Comment by jlurien** [comment]
Before merging this in main we have to set versions to wip @maxl2287 @bigludo7 I'm ok with the rest of content.

**Review by bigludo7 — DISMISSED** [review]
LGTM

**Review by bigludo7 — COMMENTED** [review]
Hi @maxl2287 
Line 86 should have wip as version as well no?

**Comment by jlurien** [comment]
> Hi @maxl2287 Line 86 should have wip as version as well no?

Yes, please @dfischer-tech could you change it to:

 - url: "{apiRoot}/geofencing-subscriptions/vwip"

**Comment by maxl2287** [comment]
@jlurien - I did it now for @dfischer-tech. 
Can be now approved from my side.

**Review by maxl2287 — APPROVED** [review]
LGTM ✅

**Review by bigludo7 — APPROVED** [review]
LGTM

**Review by jlurien — APPROVED** [review]
LGTM

**Pull request merged**